cryptofeed/exchanges/ascendex.py

```diff
@@ -24,12 +24,25 @@
 
 class AscendEX(Feed):
     id = ASCENDEX
-    websocket_endpoints = [WebsocketEndpoint('wss://ascendex.com/1/api/pro/v1/stream')]
-    rest_endpoints = [RestEndpoint('https://ascendex.com', routes=Routes('/api/pro/v1/products'))]
+    rest_endpoints = [RestEndpoint(
+        'https://ascendex.com',
+        routes=Routes('/api/pro/v1/products'),
+        sandbox='https://api-test.ascendex-sandbox.com'
+    )]
     websocket_channels = {
         L2_BOOK: 'depth:',
         TRADES: 'trades:',
     }
+    # Docs, https://ascendex.github.io/ascendex-pro-api/#websocket-authentication
+    # noinspection PyTypeChecker
+    websocket_endpoints = [WebsocketEndpoint(
+        'wss://ascendex.com/1/api/pro/v1/stream',
+        channel_filter=(
+            websocket_channels[L2_BOOK],
+            websocket_channels[TRADES],
```

Comment:
why did you add a channel filter here? Seems like that would only be needed if you had added ascendex futures to this class as a second websocket endpoint.

Reply:
As they have private endpoint for private stream, it is for private channels support in the future.

See, https://ascendex.github.io/ascendex-pro-api/#websocket-request